frontend/src/pages/CompareV2.tsx

```diff
@@ -275,6 +286,22 @@ function VisualizationPanelItem(props: VisualizationPanelItemProps) {
     { staleTime: Infinity },
   );
 
+  useEffect(() => {
+    if (isLoading) {
+      return;
+    }
+
+    if (isError) {
+      (async function() {
+        const updatedMessage = await errorToMessage(error);
+        setErrorMessage(updatedMessage);
+        setShowError(true);
+      })();
+    } else {
+      setShowError(false);
```

Comment:
Is `ShowError` false by default? Is this to make sure the error banner disappears when the page successfully fetches the data?

Reply:
Yes, show error is initialized to `false`, and then is only set to `true` if there is an error found when trying to fetch the viewer configs.